crates/orderbook/src/database/auctions.rs

```diff
@@ -15,8 +12,94 @@ impl super::Postgres {
             Some(inner) => inner,
             None => return Ok(None),
         };
-        let auction: Auction = serde_json::from_value(json)?;
-        let auction = AuctionWithId { id, auction };
+        let auction: dto::Auction = serde_json::from_value(json)?;
+        let auction = dto::AuctionWithId { id, auction };
         Ok(Some(auction))
     }
 }
+
+pub mod dto {
```

Comment:
Should we start creating a high level dto model for the orderbook (something like crates/orderbook/src/dto) so that we have a place where we can progressively move the model::* types into?

Reply:
Indeed. I moved it to high level folder.